dev-app.js

```diff
@@ -10,6 +10,7 @@ const port = process.env.PORT;
 
 function createAuthorizeRequest() {
   const vtr = process.env.VTR ? "vtr=" + encodeURI(process.env.VTR) : "";
+  const idToken = process.env.SUPPORT_REAUTHENTICATION ? "&id_token_hint=" + process.env.ID_TOKEN_VALUE : ""
```

Comment:
I'm not sure if we want this always set when the feature flag is turned on, as it'll make every journey a re-auth journey. How are you envisioning toggling this on and off when the feature flag is on?